packages/react-components/src/molecules/UserProfileLabText.tsx

```diff
@@ -0,0 +1,17 @@
+import { UserResponse } from '@asap-hub/model';
+import { Ellipsis } from '../atoms';
+import { getUniqueCommaStringWithSuffix } from '../utils';
+
+type UserProfileLabTextProps = Partial<Pick<UserResponse, 'labs'>>;
+
+const UserProfileLabText: React.FC<UserProfileLabTextProps> = ({
```

Comment:
I don't like this component as it can be replaced with one line of code. I think it's already caused problems like introducing Ellipsis where it shouldn't be (see below).
`<Ellipsis>{getUniqueCommaStringWithSuffix(labs.map(({name}) => name),'Lab')}</Ellipsis>`